test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- [ ] Ooops!!! This file was not actually ignored by git😃. Kindly remove the / since it is a file not directory. 

> test.md